IFC4x3/ModelViews/General Usage/DocModelView.xml

```diff
@@ -7984,23 +7986,28 @@ The material is attached by the _RelatingMaterial_ attribute on the [IfcRelAssoc
 						<DocTemplateItem UniqueId="3967f405-c591-437b-a6ee-6b8a2623b6ba" />
 					</Items>
 				</DocTemplateUsage>
+				<DocTemplateUsage Name="Borehole Assignment" UniqueId="b1a6ef95-8767-456a-99f6-fcbfabe379af">
+					<Documentation>An _IfcGeoScienceFeature_ may be assigned to an _IfcBorehole_ where each _IfcGeoScienceFeature_ represents one interpreted homogeneous unit (e.g. an interpreted material) in the _IfcBorehole_.</Documentation>
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Assignment_to_Product_2Gn4FzSQ96MQoPL52lkLa4" />
+				</DocTemplateUsage>
 			</Concepts>
 			<ApplicableTemplate xsi:type="DocTemplateDefinition" UniqueId="7e0ad082-f4df-42ca-aba6-dfe477bbfabd" Type="IfcGeoScienceElement" />
 		</DocConceptRoot>
 		<DocConceptRoot Name="IfcGeoScienceObservation" UniqueId="99efba22-85a4-44e1-9c6b-43c34560aa44">
 			<ApplicableEntity xsi:type="DocEntity" xsi:nil="true" href="IfcGeoScienceObservation" />
 			<Concepts>
 				<DocTemplateUsage Name="Observation Composition" UniqueId="f9189054-b6de-41d0-8c9e-b9b65480502e">
+					<Documentation>An _IfcGeoScienceObservation may decompose another _IfcGeoScienceObservation_. This may be the case e.g. when several different types of tests have been performed in an _IfcBorehole_ at different depths and each type of test is decomposed into sub-tests at different depth ranges.</Documentation>
```

Comment:
```suggestion
					<Documentation>An _IfcGeoScienceObservation_ may decompose another _IfcGeoScienceObservation_. This may be the case e.g. when several different types of tests have been performed in an _IfcBorehole_ at different depths and each type of test is decomposed into sub-tests at different depth ranges.</Documentation>
```

Reply:
Fixed in [5bb93ae](https://github.com/bSI-InfraRoom/IFC-Specification/pull/870/commits/5bb93aed8f9a559073a1c797d067ba693bf59d95)